### DISCUSSION TITLE
DouglasPeuckerLineSimplifier

### DISCUSSION THREAD

**Discussion by ericbowman-tomtom** [locationtech/jts#849] [Q&A] [answered]
Why is [DouglasPeuckerLineSimplifier](https://github.com/locationtech/jts/blob/da3923d55b02bf2539ab336d1685e1173c9f91a2/modules/core/src/main/java/org/locationtech/jts/simplify/DouglasPeuckerLineSimplifier.java#L25) not a public class?

**Comment by dr-jts** [comment, marked as answer]
It's just intended as a way of modularizing code used by the public API of `DouglasPeuckerSimplifier`.  

Generally speaking, we try and keep the public API as small as possible, to make ongoing code evolution easier.
